src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+const arrList = document.querySelectorAll('li');
+const arrDate = [];
+
+function sortList(list) {
+  for (let i = 0; i < arrList.length; i++) {
+    arrDate.push({
+      name: arrList[i].textContent,
+      position: arrList[i].dataset.position,
+      salary: arrList[i].dataset.salary,
+      age: arrList[i].dataset.age,
+    });
+  }
+
+  return arrDate;
+}
```

Comment:
The `sortList` function is defined with a `list` parameter, but it is not used. Instead, the function relies on the global `arrList` variable. Consider using the `list` parameter to make the function more flexible and reusable.